HACKERS

```diff
@@ -134,29 +134,56 @@ object will not get missing symbols when loaded.
 It also means you have to check for and skip tests that aren't relevant to this
 version.  The recommended way to do this is like:
 
- if (ivers($]) < ivers(5.6.2)) {
+ if (ivers($]) < ivers(5.6)) {           # No quotes needed
```

Comment:
is 5.6 the correct thing to do instead of the previously used 5.6.2?

Reply:
it's just documentation.  And quotes are never needed with a single dot.  double-dotted is explained in the vicinity of this